src/test/suite/didact.test.ts

```diff
@@ -88,6 +93,61 @@ suite('Didact test suite', () => {
 		}
 	});
 
+	test('Scaffold new project, test for open: true file', async function () {
+		try {
+			await commandHandler.processInputs(testScaffoldOpen).then( async () => {
+				// test to make sure the groovy file, set to open: true in json, actually opens
+				try {
+					await waitUntil(() => {
+						return findEditorForFile(`simple.groovy`);
+					}, EDITOR_OPENED_TIMEOUT, 1000);
+				} catch (error) {
+					assert.fail(`simple.groovy has not been opened in editor`);
+				}
+			});
+		} catch (error) {
+			assert.fail(error);
+		}
+	});
+
+	test('Scaffold new project, test for standard file (no open)', async function () {
+		try {
+			await commandHandler.processInputs(testScaffoldOpen).then( async () => {
+				// test to make sure the other file, set to open: false in json, does not open
```

Comment:
the comment is not accurate. it is not open: false, it is without open value set.